ambry-store/src/main/java/com.github.ambry.store/BlobStore.java

```diff
@@ -227,7 +285,7 @@ public void put(MessageWriteSet messageSetToWrite) throws StoreException {
       Offset indexEndOffsetBeforeCheck = index.getCurrentEndOffset();
       MessageWriteSetStateInStore state = checkWriteSetStateInStore(messageSetToWrite, null);
       if (state == MessageWriteSetStateInStore.ALL_ABSENT) {
-        synchronized (lock) {
+        synchronized (logWriteLock) {
```

Comment:
+1 on naming this in a more meaningful way. This is not just a lock against writes to the log though strictly speaking. More like "storeWriteLock"?

Reply:
The comment that was describing it before I took it out said, "/* A lock that prevents concurrent writes to the log */" So that was inaccurate?

Reply:
Went ahead and changed it to 'storeWriteLock'